src/runtimes/wasi_cli/errors.rs

```diff
@@ -4,9 +4,8 @@ pub type Result<T> = std::result::Result<T, WasiRuntimeError>;
 
 #[derive(Error, Debug)]
 pub enum WasiRuntimeError {
-    #[error("program exited with code {code:?}; stderr set to '{stderr}', error: '{error}'")]
+    #[error("{stderr}")]
```

Comment:
removed Code as we are not using it to build the display string, however the return code is used in a debug log (see below)